app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self,
+                 name: str,
+                 age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(p_list: list) -> list:
+    return_list = []
+
+    for i, prs in enumerate(p_list):
+        return_list.append(Person(prs["name"], prs["age"]))
+        if "wife" in prs and prs["wife"] is not None:
+            return_list[i].wife = prs["wife"]
+        if "husband" in prs and prs["husband"] is not None:
+            return_list[i].husband = prs["husband"]
```

Comment:
You can simplify your code here with using method `get` of `dict`
```suggestion
        if prs.get("wife"):
            return_list[i].wife = prs["wife"]
        if prs.get("husband"):
            return_list[i].husband = prs["husband"]
```

Reply:
Grate solution!

Reply:
Done.